app/_locales/en/messages.json

```diff
@@ -3777,6 +3778,9 @@
   "replace": {
     "message": "replace"
   },
+  "reportProblem": {
+    "message": "Report a problem"
```

Comment:
based on the comments, looks like we want to update this to  `Report an issue` 

ref: https://github.com/MetaMask/metamask-extension/pull/22274#issuecomment-1861344879